packages/cli/src/tools/config/index.js

```diff
@@ -196,7 +196,7 @@ function loadConfig(projectRoot: string = process.cwd()): ConfigT {
         .join(
           '\n',
         )}\nPlease notify their maintainers about it. You can find more details at ${chalk.dim.underline(
-        'https://react-native-community/cli/docs/configuration.md#migration-guide',
+        'https://github.com/react-native-community/cli/blob/master/docs/configuration.md',
```

Comment:
```suggestion
        'https://github.com/react-native-community/cli/blob/master/docs/configuration.md#migration-guide',
```

Reply:
We should point to the section with migrate guide.

Reply:
Oops, done.